### PR TITLE
Add mne-bids-pipeline

### PR DESCRIPTION
**Pull request by larsoner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/mne-bids-pipeline/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+about:
+  home: https://pypi.org/project/mne-bids-pipeline/
+  summary: A standardized processing pipeline for your MEG and EEG data
+  license: BSD-3-Clause
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - larsoner
+    - hoechenberger
```
Comment (larsoner) [inline recipes/mne-bids-pipeline/meta.yaml:66]: @hoechenberger can you confirm you're willing to co-maintain this recipe?

**Comment by hoechenberger** [comment]
I agree to co-maintain this package / feedstock.

File: recipes/mne-bids-pipeline/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  run:
+    - autoreject
+    - bokeh <3
+    - coloredlogs
+    - dask-core
+    - filelock
+    - joblib >=0.14
+    - json_tricks
+    - jupyter-server-proxy
+    - mne
```
Comment (hoechenberger) [inline recipes/mne-bids-pipeline/meta.yaml:32]: We should add lower bounds on the version numbers

File: recipes/mne-bids-pipeline/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+requirements:
+  host:
+    - pip
+    - setuptools >=65.0
+    - python >=3.8
+  run:
+    - autoreject
+    - bokeh <3
+    - coloredlogs
+    - dask-core
```
Comment (hoechenberger) [inline recipes/mne-bids-pipeline/meta.yaml:29]: I think we also need `distributed` but I'm not entirely sure, could you check please?

Comment (larsoner) [reply recipes/mne-bids-pipeline/meta.yaml:29]: I'll just replace with `dask` which takes care of bokeh + distributed

**Comment by larsoner** [comment]
Argh, `mne_bids_pipeline --version` reports `0.0.0`. I'll try installing `setuptools_scm` but I doubt this will fix it. There is no tag to look up in the `sdist` that is used.

We can probably fix it using what `codespell` does

https://github.com/codespell-project/codespell/blob/cef86165e57081ad8b7cdc27dab72e149b8374b6/pyproject.toml#L66-L67

**Comment by hoechenberger** [comment]
> There is no tag to look up in the sdist that is used.

I tested exactly this locally a few weeks ago and there was indeed a version number in the sdist I had created via `python-m build` 🤔

File: recipes/mne-bids-pipeline/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    - packaging
+    - pandas
+    - psutil
+    - python >=3.8
+    - python-picard
+    - pyvista
+    - pyvistaqt
+    - qtpy
+    - scikit-learn
+    - seaborn
```
Comment (hoechenberger) [inline recipes/mne-bids-pipeline/meta.yaml:46]:
```suggestion
    - seaborn
    - setuptools  # https://github.com/pypa/setuptools_scm/#retrieving-package-version-at-runtime
```

**Comment by hoechenberger** [comment]
> > There is no tag to look up in the sdist that is used.
> 
> I tested exactly this locally a few weeks ago and there was indeed a version number in the sdist I had created via `python -m build` 🤔
> 
> In any case, depending on setuptools-scm shouldn't be necessary I think 🧐
> 
> But cannot check right now

I can try to reproduce this later today

**Comment by hoechenberger** [comment]
@larsoner It appears the correct version number is being reported now

**Review by dopplershift — APPROVED** [review]

**Pull request merged**